src/ott/problems/linear/potentials.py

```diff
@@ -157,6 +161,141 @@ def tree_unflatten(
   ) -> "DualPotentials":
     return cls(*children, **aux_data)
 
+  def plot_ot_map(
+      self,
+      source: jnp.ndarray,
+      target: jnp.ndarray,
+      inverse: bool = False,
+      alpha: float = 0.5,
+      ax: Optional[matplotlib.axes.Axes] = None,
+      **kwargs: Any,
+  ) -> Tuple[matplotlib.figure.Figure, matplotlib.axes.Axes]:
+    """Plot data and learned optimal transport map.
+
+    Args:
+      learned_potentials: the potentials object for the map
+      source: samples from the source measure
+      target: samples from the target measure
+      inverse: if the inverse map from the potentials
+        should be used
+      ax: axis to add the plot to
+      alpha: transparency of the scatter plot points
+      kwargs: additional kwargs passed into ``scatter``
+
+    Returns: matplotlib figure and axis with the plots
+    """
+    if ax is None:
+      fig = plt.figure(facecolor="white")
+      ax = fig.add_subplot(111)
+    else:
+      fig = ax.get_figure()
+
+    # plot the source and target samples
+    if not inverse:
+      label_transport = r"$\nabla f(source)$"
+      source_color, target_color = "#1A254B", "#A7BED3"
+    else:
+      label_transport = r"$\nabla g(target)$"
+      source_color, target_color = "#A7BED3", "#1A254B"
+
+    ax.scatter(
+        source[:, 0],
+        source[:, 1],
+        color=source_color,
+        alpha=alpha,
+        label='source',
+        **kwargs,
+    )
+    ax.scatter(
+        target[:, 0],
+        target[:, 1],
+        color=target_color,
+        alpha=alpha,
+        label='target',
+        **kwargs,
+    )
+
+    # plot the transported samples
+    base_samples = source if not inverse else target
+    transported_samples = self.transport(base_samples, forward=not inverse)
+    ax.scatter(
+        transported_samples[:, 0],
+        transported_samples[:, 1],
+        color="#F2545B",
+        alpha=alpha,
+        label=label_transport,
+        **kwargs,
+    )
+
+    for i in range(base_samples.shape[0]):
+      ax.arrow(
+          base_samples[i, 0],
+          base_samples[i, 1],
+          transported_samples[i, 0] - base_samples[i, 0],
+          transported_samples[i, 1] - base_samples[i, 1],
+          color=[0.5, 0.5, 1],
+          alpha=0.3
+      )
+
+    ax.legend()
+    return fig, ax
+
+  def plot_potential(
+      self,
+      inverse: bool = False,
+      alpha: float = 0.05,
+      ax: Optional[matplotlib.axes.Axes] = None,
+      bounds: Tuple[float] = (-6, 6, -6, 6),
```

Comment:
Think it's a bit nicer either to have 2 arguments ({x,y}_bounds), or have `)(-6, 6), (-6, 6))` instead of a flat tuple.

Reply:
Updated to `{x,y}_bounds`